spec/features/idv/analytics_spec.rb

```diff
@@ -295,6 +305,7 @@
     }.compact
   end
 
+  # TODO: Add ["IdV: doc auth link_sent visited", "IdV: doc auth capture_complete visited", "IdV: doc auth link_sent submitted"]
```

Comment:
Thank you for noting these. Can you add a task to do this work, and we'll make sure someone else can share the pain

Reply:
New ticket: [LG-15308](https://cm-jira.usa.gov/browse/LG-15308)